wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java

```diff
@@ -98,7 +101,6 @@ private Object readDeclaration(String documentation, Context context) {
       if (!context.permitsPackage()) throw reader.unexpected(location, "'package' in " + context);
       if (packageName != null) throw reader.unexpected(location, "too many package names");
       packageName = reader.readName();
-      fileBuilder.packageName(packageName);
```

Comment:
Is it safe to just delete this line? Probably should do `location.copy()` and replace the package name?

Reply:
See the other changes in this file.

Reply:
Ah, got it, nvm.